docs/how-to-guides/themes/theme-json.md

```diff
@@ -1179,12 +1179,7 @@ As a result of this change, it’s now the block author and theme author’s res
 
 ### What is blockGap and how can I use it?
 
-blockGap adjusts the vertical margin, or gap, between blocks.
-It is also used for margins between inner blocks in columns, buttons, and social icons.
-In the editor, the control for the blockGap is called Block spacing, located in the Dimensions panel.
-
-The value you define for the blockGap style uses a CSS property, a preset, named `--wp--style--block-gap`.
-The default value is 2em.
```

Comment:
I've removed this line about the default value, as it will be different per theme, and the core `theme.json` shipped in the editor has a different default (currently `24px`) so specifying the particular value in the docs could cause confusion. Instead, at the bottom of this changeset I've added a line that the root `blockGap` value will be output to the CSS variable.

Reply:
I think we need to make it even clearer what is supported by core "block editor" in 6.0 and what is supported in the active Gutenberg plugin.

Reply:
it should not block this PR though -I think we need to reconsider what information should go into the theme developer handbook and what should be on this page and even split it into subpages.

Reply:
Thanks for taking a look @carolinan! Yes, it'd be great to explore options there in follow-ups, so that we have some neater areas for expanding on how different functionality works, particularly as the features keep growing / becoming more complex.

I'd be keen to do a follow-up PR at some point to expand on the current state of Layout support, but it'd probably need to be specific to "here's how it works in Gutenberg right now" so it'd be good to figure out how to neatly flag / present that in a way that's approachable for folks who regularly use these docs. Let me know if you have any suggestions and I'm happy to have a go at a follow-up PR with some extra context next week.